web.config

```
@@ -7,7 +7,7 @@
     <httpPlatform stdoutLogEnabled="true"
                   stdoutLogFile="%HOME%\LogFiles\stdout"
                   processPath="%JAVA_HOME%\bin\java.exe"
-                  arguments="-javaagent:&quot;%HOME%\site\wwwroot\applicationinsights-agent-2.3.1.jar&quot; -Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\ethos-repl-docmosis-service.jar&quot;">
+                  arguments="-Xms1024M -Xmx2048M -javaagent:&quot;%HOME%\site\wwwroot\applicationinsights-agent-2.4.1.jar&quot; -Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\core-case-data.jar&quot;">
```

the jar in lib should also be updated
```suggestion
                  arguments="-Xms1024M -Xmx2048M -javaagent:&quot;%HOME%\site\wwwroot\applicationinsights-agent-2.5.0.jar&quot; -Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\core-case-data.jar&quot;">
```